app/src/main/cpp/skyline/applet/error_applet.h

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2022 Skyline Team and Contributors (https://github.com/skyline-emu/)
+// Copyright © 2020 Ryujinx Team and Contributors (https://github.com/ryujinx/)
+
+#pragma once
+
+#include <services/am/applet/IApplet.h>
+#include "common/language.h"
+
+namespace skyline::applet {
+    /**
+     * @brief The ErrorApplet class is used to show errors to the user
+     * @url https://switchbrew.org/wiki/Error_Applet
+     * @url https://switchbrew.org/wiki/Error_applet
+     */
+    class ErrorApplet : public service::am::IApplet, service::am::EnableNormalDataQueue {
+      private:
+        enum class ErrorType : u8 {
+            ErrorCommonArg,
+            SystemErrorArg,
+            ApplicationErrorArg,
+            ErrorEulaArg,
+            ErrorPctlArg,
+            ErrorRecordArg,
+            SystemUpdateEulaArg = 8
+        };
```

Comment:
I'd recommend setting every member of `ErrorType`, this is the general practice with guest enums.